cmd/lockinfo-handlers_test.go

```diff
@@ -18,27 +18,38 @@ package cmd
 
 import (
 	"fmt"
+	"os"
 	"testing"
 	"time"
 )
 
 // TestListLocksInfo - Test for listLocksInfo.
 func TestListLocksInfo(t *testing.T) {
-	// Initialize globalNSMutex to validate listing of lock
-	// instrumentation information.
+	// reset global variables to start afresh.
+	resetTestGlobals()
+	// Initialize minio server config.
+	rootPath, err := newTestConfig(globalMinioDefaultRegion)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(rootPath)
+	// Initializing new XL objectLayer.
+	objAPI, _, xlErr := initTestXLObjLayer()
+	if xlErr != nil {
+		t.Errorf("failed to init object layer")
```

Comment:
we should use `t.FatalF` here and exit test. We can't continue the test with a possibly `nil` `objAPI`.